### DISCUSSION TITLE
Can IAM works as a SAML IdP?

### DISCUSSION THREAD

**Discussion by zhangxt-ihep** [indigo-iam/iam#591] [Q&A] [answered]
Hello,

I have some service only accept SAML assertions to get authorization. I think IAM can offer JWT as authorization tokens, but can it works as a SAML IdP so that my SAML-only service can get authorization?

Thanks!
Xuantong

**Comment by rmiccoli** [comment, marked as answer]
Hi Xuantong,

Indigo IAM is an OpenId Connect Provider that can also act as a SAML SP, but not as a SAML IdP. For this reason, you need an OIDC-to-SAML proxy to integrate IAM with your SAML-only service. 
In the context of the ESCAPE project a [SATOSA Proxy](https://github.com/IdentityPython/SATOSA) has been chosen to satisfy a similar use case.

Roberta
